bin/prep_ap_pipe.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# A script to set things up to run ap_pipe via slurm on some DECam data.
+#
+# First, run this script:
+#       $ bash prep_ap_pipe.sh $1=rerun $2=repo $3=calib $4=template $5=filter
+# e.g., $ bash prep_ap_pipe.sh slurm1 . calibdir templatedir g
+#
+# (Note each argument is optional and is set with defaults below)
```

Comment:
Because you're using positional arguments, if you don't set one it will mess up the others: it's all or nothing.  This is also annoying because you can't just change one and keep the others as defaults, for instance.

One option would be to write a parser that let you give arguments like --repo x, etc. (see http://linuxcommand.org/lc3_wss0120.php).  Another option would be to declare environment variables outside the script (e.g., `export FILTERNAME=g`) and detect if they are set or unset.